Extension/src/githubAPI.ts

```diff
@@ -124,6 +124,11 @@ export interface BuildInfo {
     name: string;
 }
 
+/**
+ * Set this hook to true to force an update, to test the update flow.
+ */
+let testUpdateFlow: boolean = false;
```

Comment:
how is this different from `testingInsidersVsixInstall`?  Do we need both variables?

Reply:
Removed